mirdata/datasets/filosax.py

```diff
@@ -0,0 +1,512 @@
+    def name(self):
+        """The track's name
+
+        Returns:
+            * str - track name
+
+        """
+        return self.annotation["file_metadata"]["title"]
+
+    @property
+    def duration(self):
+        """The track's duration
+
+        Returns:
+            * float - track duration (in seconds)
+
+        """
+        return self.annotation["file_metadata"]["duration"]
+
+    @property
+    def beats(self):
+        """The times of downbeats and chord changes
+
+        Returns:
+            * SortedKeyList [Observation(time, duration, value)] - timestamp, duration (seconds), beat
+
+        """
+        return self.annotation.search(namespace="beat")[0]["data"]
+
+    @property
+    def chords(self):
+        """The times and values of chord changes
+
+        Returns:
+            * SortedKeyList [Observation(time, duration, value)] - timestamp, duration (seconds), chord symbol
+
+        """
+        return self.annotation.search(namespace="chord")[0]["data"]
+
+    @property
+    def segments(self):
+        """The times of segment changes (values are 'head', 'written solo', 'improvised solo')
+        Returns:
+            * SortedKeyList [Observation(time, duration, value)] - timestamp, duration (seconds), beat
+
+        """
+        return self.annotation.search(namespace="segment_open")[0]["data"]
+
+    @property
+    def bass_drums(self):
+        """The associated bass/drums track
+        Returns:
+            * Track
+
+        """
+        return self.tracks[self.mtrack_id + "_bass_drums"]
+
+    @property
+    def piano_drums(self):
+        """The associated piano/drums track
+        Returns:
+            * Track
+
+        """
+        return self.tracks[self.mtrack_id + "_piano_drums"]
+
+    @property
+    def sax(self):
+        """The associated sax tracks (1-5)
+        Returns:
+            * [Track]
+
+        """
+        return [self.tracks["%s_sax_%d" % (self.mtrack_id, n)] for n in [1, 2, 3, 4, 5]]
+
+    def to_jams(self):
+        """Jams: the track's data in jams format"""
+        return self.annotation
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
+    """Load a Filosax audio file.
+
+    Args:
+        fhandle (str or file-like): path or file-like object pointing to an audio file
+
+    Returns:
+        * np.ndarray - the audio signal
+        * float - The sample rate of the audio file
+
+    """
+    return librosa.load(fhandle, sr=None, mono=True)
+
+
+@io.coerce_to_string_io
+def load_annotation(fhandle: TextIO) -> List[Note]:
+    """Load a Filosax annotation file.
+
+    Args:
+        fhandle (str or file-like): path or file-like object pointing to an audio file
+
+    Returns:
+        * List[Note]: an ordered list of Note objects
+
+    """
+    note_dict = json.load(fhandle)["notes"]
+    return [Note(n) for n in note_dict]
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """The Filosax dataset"""
```

Comment:
If you take a look at the documentation link I attached at the end of the review, you can see that the documentation of the dataset class for `filosax` looks badly formatted.

Reply:
Sorry, @dave-foster! I think [this link](https://mirdata--536.org.readthedocs.build/en/536/source/mirdata.html) should work. In case it does not work, to see the docs rendered in your PR you need to get to [the doc most recent builds](https://readthedocs.org/projects/mirdata/builds/) and click on the latest build generated by your PR (#536), and then click on `View docs` in the top right side of the page. That will redirect you to the docs your PR is rendering.

Reply:
Thanks @genisplaja - all docs now looking okay